### PR TITLE
feat: add events media

### PR DESCRIPTION
**Pull request by simboonlong**
- https://www.pivotaltracker.com/story/show/177376008
- re-order latest to oldest
- will do the cms story in another pr

### CODE REVIEW

File: src/pages/media.tsx [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
+    title: "Singapore leading the way in promoting digital trade",
+    date: "3 Feb 21",
+    url: "https://www.businesstimes.com.sg/opinion/singapore-leading-the-way-in-promoting-digital-trade ",
   },
   {
     title:
-      "World's Largest Business Organisation Joins Singapore Government and Major Industry Partners to Drive Global Digitalisation",
-    date: "22 Jan 2020",
+      "Electronic Transactions Act Amended To Facilitate Electronic Transactions, Providing Convenience And Strengthening Singapore’s Trade Competitiveness In The Digital Economy",
+    date: "1 Feb 21",
```
Comment (cavacado) [inline src/pages/media.tsx:22]: date convention varies from resources
here its 1 Feb 21
below is 5 Oct 2020

File: src/pages/media.tsx [hunk 2]
```diff
@@ -39,50 +42,79 @@ const pressReleases: Resource[] = [
...
     description:
-      "This event by BAFT and R3 explored different blockchain-based digital trade solutions and how global trade participants can use these solutions towards challenges in the global trade ecosystem.",
+      "Designed for businesses of all sizes, learn how you can digitise your supply chains to save cost, improve efficiency and overall profitability by using the Trames platform.",
     resource: {
-      dateTime: "2 July | 9-10pm SGT",
-      watchLink: "https://www.youtube.com/watch?v=iRmaQV9HERw",
-      eventLink: "https://www.baft.org/baft-education/e-learning/past-webinars",
-      eventSlides:
-        "https://baft.org/docs/default-source/past-webinars/webinar-the-journey-to-paperless-trade.pdf?sfvrsn=6e9a20e1_2",
+      dateTime: "8 Apr 2021, 3-4pm (GMT+8)",
```
Comment (cavacado) [inline src/pages/media.tsx]: here is even worse.
dateTime, then some are just dates.

**Review by cavacado — APPROVED** [review]
even though there are outstanding comments,
i have talked to @simboonlong about them and he will handle them in the 
next pr concerning moving the events/resources to CMS

File: src/pages/media.tsx (hunk 1 above)
Comment (simboonlong) [reply src/pages/media.tsx:22]: updated to follow D MMM YYYY for now

File: src/pages/media.tsx (hunk 2 above)
Comment (simboonlong) [reply src/pages/media.tsx]: separated time from date since not all events has time, will figure out on timezone later in the cms pr

**Pull request merged**